spec/challenge_spec.rb

```diff
@@ -17,8 +20,6 @@
 
   describe '#each' do
     it 'stores minimal state' do
-      require 'binding_of_caller'
```

Comment:
Any particular reason moving dependencies away from their point of injection is preferable?

If this were a class, I could be persuaded. This is a test. A spec file can get very long.